packages/teams-js/src/internal/communication.ts

```diff
@@ -124,6 +124,14 @@ export function uninitializeCommunication(): void {
   CommunicationPrivate.promiseCallbacks = {};
 }
 
+/**
+ * @internal
```

Comment:
I know this file doesn't do good at documenting functions, but brief descriptions of what these functions are for would be really great!

Reply:
I tried my best understand on those functions and write some small comments on APIs I added. Please take a look. And also do let me know if you want me to add for the others.